### DISCUSSION TITLE
[FIXED] Difference between „Modal“ and „Overlay“

### DISCUSSION THREAD

**Discussion by Nevu** [ingrammicro/puzzle-publisher#42] [General]
Hi,
What are the differences between “Overlay„ and „Modal“ (and „Regular Page“)?
Can't found in description.

Regards

**Comment by MaxBazarov** [comment]
"A modal window creates a mode that disables the main window but keeps it visible, with the modal window as a child window in front of it."
An overlay just shows an additional layer over the main window..It doesn't block interaction with the main window.
